coreneuron/nrniv/global_vars.cpp

```diff
@@ -29,7 +29,7 @@ void hoc_register_var(DoubScal* ds, DoubVec* dv, VoidFunc*) {
     }
 }
 
-void set_globals(const char* path) {
+void set_globals(const char* path, bool cli_global_seed, int cli_global_seed_value) {
```

Comment:
@fouriaux : I think we see we will parse the option as long and pass command line argument as uint, isn't it? (to avoid overflow)